includes/degree-functions.php

```diff
@@ -3,6 +3,378 @@
+					<?php if ( $promo_image ) : ?>
+					<img src="<?php echo $promo_image; ?>" class="img-fluid p-4" role="img">
+					<?php endif; ?>
+				</div>
+
+			</div>
+		</div>
+	</div>
+
+	<?php
+	return ob_get_clean();
+
+ }
+
+
+
+ /**
+ * Returns HTML for the degree description section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @return string HTML markup
+  *
+  */
+  function get_degree_description_modern_layout() {
+	$degree_description_modern_layout = get_field( 'degree_description_modern_layout' );
+
+	if( empty( $degree_description_modern_layout ) ) return '';
+
+	ob_start();
+	?>
+	<div class="container py-lg-3">
+		<div class="row my-lg-3">
+			<?php echo $degree_description_modern_layout; ?>
+		</div>
+	</div>
+	<?php
+
+	return ob_get_clean();
+  }
+
+
+
+  /**
+   * Returns HTML for the application deadline section.
+   *
+   * @since 3.4.0
+   * @author RJ Bruneel
+   * @return string HTML markup
+   *
+   */
+   function get_degree_application_deadline_modern_layout() {
+	 $application_deadline_first = get_field( 'application_deadline_first' );
+	 $application_deadline_second = get_field( 'application_deadline_second' );
+
+	 if( empty( $application_deadline_first ) && empty( $application_deadline_second ) ) return '';
+ 
+	 ob_start();
+	 ?>
+	 <div class="bg-default">
+		<div class="container py-lg-3">
+			<div class="row my-lg-3">
+				<div class="col text-center text-uppercase">
+					<?php echo $application_deadline_first; ?><br>
+					<?php echo $application_deadline_second; ?>
+				</div>
+			</div>
+		</div>
+	</div>
+	 <?php
+ 
+	 return ob_get_clean();
+   }
+
+
+
+ /**
+  * Returns HTML for the course overview section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @return string HTML markup
+  *
+  */
+  function get_degree_course_overview_modern_layout() {
+	$course_overview = get_field( 'course_overview' );
+
+	if( empty( $course_overview ) ) return '';
+
+	ob_start();
+	?>
+	<div class="bg-faded">
+		<div class="container py-lg-3">
+			<div class="row my-lg-3">
+				<?php echo $course_overview; ?>
+			</div>
+		</div>
+	</div>
+	<?php
+
+	return ob_get_clean();
+  }
+
+
+
+/**
+  * Returns HTML for the degree skills and careers section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @return string HTML markup
+  *
+  */
+ function get_degree_skills_career_modern_layout() {
+	$skills_heading = get_field( 'skills_heading' );
+	$skills_content = get_field( 'skills_content' );
+	$careers_heading = get_field( 'careers_heading' );
+	$careers_content = get_field( 'careers_content' );
+
+	if( empty( $skills_content ) && empty( $careers_content ) ) return '';
+
+	ob_start();
+	?>
+	<div class="bg-inverse">
+		<div class="container">
+			<div class="row py-lg-5">
+				<div class="col-12">
+					<?php if( $skills_heading ) : ?>
+						<h2 class="font-condensed text-primary text-uppercase mb-4"><?php echo $skills_heading; ?></h2>
+					<?php endif; ?>
+				</div>
+				<div class="col-lg-8 py-lg-3">
```

Comment:
Should we be displaying this column (empty space) if `$skills_content` is empty?